data/skeletons/lalr1.cc

```diff
@@ -235,7 +235,29 @@ m4_define([b4_shared_declarations],
     void error (const syntax_error& err);
 
 ]b4_token_constructor_define[
+]b4_parse_error_bmatch([verbose\|detailed\|custom], [[
+    class Context {
+      public:
+        Context(const ]b4_parser_class[& yyparser,
+                     symbol_type yyla)
+          : yyparser(yyparser),
+            yyla(yyla)
```

Comment:
Space paren (3 times).  The indentation of the second argument is weird.

Reply:
Also, I prefer the "operator" to start the next line rather than end the previous one:
``` C++
          : yyparser(yyparser)
          , yyla(yyla)
```